end_to_end/1-data-prep-e2e.ipynb

```diff
@@ -0,0 +1,1401 @@
+{
```

Comment:
<p>This section should be in the 0th notebook. In this notebook, it would be better to only focus on the data prep, processing, and using DW and Feature Store.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1974/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>agreed</p>